packages/airnode-deployer/terraform/airnode/aws/main.tf

```diff
@@ -76,7 +76,7 @@ module "testApi" {
   handler            = "index.testApi"
   source_dir         = var.handler_dir
   memory_size        = 256
-  timeout            = 30
+  timeout            = 15
```

Comment:
I set `testApi` handler timeout (both AWS and GCP) to 15 because the only thing it does is call the `callAPI` handler which has a timeout of 10. So, anything higher is unnecessary.